src/neuroglancer/sliceview/decode_jpeg_stack.ts

```diff
@@ -20,14 +20,41 @@ import {Vec3, vec3} from 'neuroglancer/util/geom';
 export function decodeJpegStack(data: Uint8Array, chunkDataSize: Vec3) {
```

Comment:
I didn't change this, yet, since it seems like it will involve a lot of file changes across the datasource folder and may be better in a separate pull request. Do you agree?